frontend/app-development/features/textEditor/TextEditor.tsx

```diff
@@ -40,15 +59,20 @@ export const TextEditorImpl = () => {
   } = useGetAppTextsByLangCodeQuery(
     {
       ...orgApp,
-      langCode: getSelectedLangCode(),
+      langCode: selectedLangCode,
     },
-    { skip: !getSelectedLangCode() }
+    { skip: !selectedLangCode }
   );
 
   const [updateLang] = useUpdateTranslationByLangCodeMutation();
   const [deleteLanguage] = useDeleteByLangCodeMutation();
   const [addLanguage] = useAddByLangCodeMutation();
 
+  const t = (key: string) => getLanguageFromKey(key, language);
```

Comment:
Hva tenker du om å ta i bruk useText-hooken? :) 
```suggestion
const t = useText()
```

Reply:
I have considered it, but it is currently defined only in the UX-editor package